include/srtp.h

```diff
@@ -1424,7 +1408,7 @@ typedef enum {
 
 typedef struct srtp_event_data_t {
   srtp_t        session;  /**< The session in which the event happend. */
-  srtp_stream_t stream;   /**< The stream in which the event happend.  */
+  uintptr_t     ssrc;     /**< The ssrc in host order of the stream in which the event happend */
```

Comment:
ok, but If the check is added to configure.in and a conditional define set in interges.h, is the change otherwise sane given the potential breakage.